Competition2018/src/test/java/competition/subsystems/lean/Commands/DecreaseSpeedCommandTest.java

```diff
@@ -0,0 +1,41 @@
+package competition.subsystems.lean.Commands;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import competition.subsystems.lean.LeanSubsystem;
+import competition.subsystems.lean.commands.DecreaseSpeedCommand;
+import xbot.common.controls.actuators.mock_adapters.MockCANTalon;
+import xbot.common.injection.BaseWPITest;
+
+public class DecreaseSpeedCommandTest extends BaseWPITest{
+	
+	LeanSubsystem lean;
+	DecreaseSpeedCommand command;
+	
+	@Override
+	public void setUp() {
+		super.setUp();
+		
+		lean = injector.getInstance(LeanSubsystem.class);
+		command = injector.getInstance(DecreaseSpeedCommand.class);
+		
+		lean.temporaryHack();
+	}
+	
+	@Test
+	public void simpleTest() {
+		command.initialize();
+		command.execute();
+	}
+	
+	@Test
+	public void checkDecreaseSpeed() {
+		command.initialize();
+		command.execute();
+		
```

Comment:
see general stuff above about asserting 0 beforehand

Reply:
used the check for zero between initialize and execute just as I did for Climber Deploy Arm Command Tests

Reply:
Which does not work because the method to change the speed in in initialize. 
So I've moved the check to before Initialize and it passes the test